document-store/src/main/java/org/hypertrace/core/documentstore/postgres/query/v1/vistors/PostgresFromTypeExpressionVisitor.java

```diff
@@ -144,7 +152,7 @@ private static String prepareTable0Query(PostgresQueryParser postgresQueryParser
   /*
   Returns the column name with double quotes if the collection is flat to prevent folding to lower-case by PG
    */
-  private String getColName(boolean isFlatCollection, String pgColumnName) {
+  private String shouldQuoteColName(boolean isFlatCollection, String pgColumnName) {
```

Comment:
The new method name seems like it would return a boolean.

Reply:
Ah! Will fix it.